pkg/gofr/datasource/kv-store/badger/badger_test.go

```diff
@@ -0,0 +1,77 @@
+package badger
+
+import (
+	"bytes"
+	"context"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+	"os"
+	"testing"
+)
+
+func setupDB(t *testing.T) *client {
+	t.Helper()
+	cl := New(Configs{DirPath: "test_badger"})
+
+	var logs []byte
+
+	ctrl := gomock.NewController(t)
+
+	cl.UseLogger(NewMockLogger(DEBUG, bytes.NewBuffer(logs)))
+	cl.UseMetrics(NewMockMetrics(ctrl))
+	cl.Connect()
+
+	t.Cleanup(func() {
+		os.RemoveAll("test_badger")
```

Comment:
This one could be removed if you do that

https://github.com/gofr-dev/gofr/pull/835#discussion_r1675505360